clang/tools/clang-sycl-linker/ClangSYCLLinker.cpp

```diff
@@ -440,9 +570,15 @@ int main(int argc, char **argv) {
   DryRun = Args.hasArg(OPT_dry_run);
   SaveTemps = Args.hasArg(OPT_save_temps);
 
-  OutputFile = "a.out";
-  if (Args.hasArg(OPT_o))
-    OutputFile = Args.getLastArgValue(OPT_o);
+  IsAOTCompileNeeded = IsIntelOffloadArch(
```

Comment:
Can we just move this inside runAOTCompile and make it a local variable?

Thanks